UI/nix-update/crypto-helpers-cocoa.mm

```diff
@@ -0,0 +1,58 @@
+#include "crypto-helpers.hpp"
+
+#import <Foundation/Foundation.h>
+#import <Security/Security.h>
+#import <Security/SecKey.h>
+
+void get_uuid(std::string &guid)
+{
+	NSString *str = [[NSUUID UUID] UUIDString];
```

Comment:
Using UUID here instead of just a random string, does that matter for the server @notr1ch?

Reply:
Actually I should probably just do what we did for YouTube OAuth and use `QRandom` as a cross-platform way of doing this, possibly with a C++ std fallback if we still want to support Qt versions < 5.10.